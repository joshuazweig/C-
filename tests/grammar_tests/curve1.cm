int main() {
    mint m1;
    mint m2;
    curve c;

    c = <m1, m2>;
}

