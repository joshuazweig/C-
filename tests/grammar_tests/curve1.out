ACCEPT
[program:
  [decls:
    [decls:]
    [fdecl:
      [typ: INT]
      ID
      LPAREN
      [formals_opt:]
      RPAREN
      LBRACE
      [combined_list:
        [combined_list:
          [combined_list:
            [combined_list: [combined_list:] [vdecl: [typ: MINT] ID SEMI]]
            [vdecl: [typ: MINT] ID SEMI]
          ]
          [vdecl: [typ: CURVE] ID SEMI]
        ]
        [stmt:
          [expr: ID ASSIGN [expr: LT [expr: ID] COMMA [expr: ID] GT]]
          SEMI
        ]
      ]
      RBRACE
    ]
  ]
  EOF
]
